README.md

```diff
@@ -45,8 +45,6 @@ Use [Composer](https://getcomposer.org/) to install Panther in your project. You
 
 ### Installing ChromeDriver and geckodriver
 
-Panther uses the WebDriver protocol to control the browser used to crawl websites.
```

Comment:
Why should we delete this? It's an important information.

Reply:
A quick Ctrl+F for "webdriver" gives me:
* It leverages the W3C's WebDriver protocol
* $client->getWebDriver()->manage()->getLog('browser');
* client->getWebDriver()->manage()->getLog('performance');
* Checking the State of the WebDriver Connection
* Use the Client::ping() method to check if the WebDriver connection is still active
* because this library uses WebDriverElement internally
* Panther is built on top of PHP WebDriver

So if you merge this, it will still be explained 7 times :-)

Reply:
It's stated only one time that it's a W3C spec and it's an important information (especially because there are several things named WebDriver).

Reply:
Sorry, I don't see your point.

> It's stated only one time that it's a W3C spec

Well, but the sentence I deleted doesn't contain that! ;-)
Besides, the "installation" section just isn't the right place to inform people what this project is all about. This should be explained at the very top, and the third sentence on the page does this very clear IMO:

> It leverages the W3C's WebDriver protocol to drive native web browsers such as Google Chrome and Firefox.